activities/intermediate/intro-to-closures/README.md

```diff
@@ -0,0 +1,2 @@
+# An introduction to closures
+The goal here is to simply get the group talking about how closures work in JS and how we can simulate the concept of private variables.
```

Comment:
What do you think about keeping a consistent style of description for these activities? See `Intro to Servers` and `Higher Order Array Functions`. These describe what's in the activity itself. Maybe something along the lines of "An introduction to closures meant to be used as an example to research and learn"? I would also probably add a description to give them a some context and maybe a link to start with, given the abstract/higher level nature of closures. Let me know your thoughts!

Reply:
I added a little to the description